kapre/tflite_compatible_stft.py

```diff
@@ -0,0 +1,234 @@
+# -*- coding: utf-8 -*-
```

Comment:
- maybe this is not necessary? 
- comment looks great. let's wrap it with `""" """`.

Reply:
Thanks for getting back to me! 

I had two ideas for the API, one using a `tflilite_compatible` bool argument for Magnitude and phase, but I thought that would get messy and complicated, hence I opted for the dimension checking. The STFT layer as it stands enforces the dimensionality (the transpose ln 193 - 198), which is why I thought it would be ok. However, I like your suggestion regarding inheritance and would be happy to put some more work into this, not a problem!

Your suggestion has the added benefit of when tensorflow adds these ops to tflite, as @Path-A pointed out, we can simply just delete the module!

I will have a go at your suggestion.

Regards

Paul

Reply:
perfect, thanks!

Reply:
I've moved all tflite layers to a separate module `time_frequency_tflite.py` all the layer inherits from corresponding layers in `time_frequency`.

the only thing that remains in `time_frequency` is the option to compute the approximate phase in the `Phase` layer. (so that you can generate training data with the approximation)

Let me know what you think

Cheers!

Paul